### PR TITLE
Feature/entities

### PR DESCRIPTION
**Pull request by jeankev**

### CODE REVIEW

File: src/main/java/com/corpoapp/corpoevent/entity/Corporation.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package com.corpoapp.corpoevent.entity;
+
+import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
```
Comment (Mal3kit) [inline src/main/java/com/corpoapp/corpoevent/entity/Corporation.java:9]: We should definitively use Lombok for getter, setter and builder

File: src/main/java/com/corpoapp/corpoevent/entity/builder/CorporationBuilder.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package com.corpoapp.corpoevent.entity.builder;
+
```
Comment (Mal3kit) [inline src/main/java/com/corpoapp/corpoevent/entity/builder/CorporationBuilder.java:2]: If we use Lombok, no need to create Builders.

File: src/main/resources/init.sql [hunk 3]
```diff
@@ -5,24 +5,30 @@ CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
...
+    duration DECIMAL
 );
 
-CREATE TABLE users(
+CREATE TABLE user(
 	mail VARCHAR(80) PRIMARY KEY NOT NULL,
 	name VARCHAR(50)
 );
 
 CREATE TABLE registered(
```
Comment (Mal3kit) [inline src/main/resources/init.sql:22]: We should verify if we need JPA annotation or not (performance gain ? structure logical ?)

File: src/test/java/com/corpoapp/corpoevent/entity/builder/EventBuilderTest.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
+package com.corpoapp.corpoevent.entity.builder;
+
```
Comment (Mal3kit) [inline src/test/java/com/corpoapp/corpoevent/entity/builder/EventBuilderTest.java:2]: We should add Container testing for integration tests (maybe in another PR ?)

**Review by Mal3kit — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties [hunk 5]
```diff
@@ -0,0 +1,7 @@
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.username=corpoapp
+quarkus.datasource.password=corpoapp
+quarkus.datasource.jdbc.driver=org.postgresql.Driver
+quarkus.datasource.jdbc.url=jdbc:postgresql://postgres:5432/corpoapp
```
Comment (Mal3kit) [inline src/main/resources/application.properties:5]: change postgres to corpo-app_postgres  (we have to put the service name of the compose here)

**Review by Mal3kit — CHANGES_REQUESTED** [review]

**Review by Mal3kit — APPROVED** [review]

**Review by tbouyenval — APPROVED** [review]

**Review by Mal3kit — APPROVED** [review]

**Pull request merged**